packages/cactus-plugin-keychain-memory/src/test/typescript/unit/plugin-keychain-memory.test.ts

```diff
@@ -18,99 +17,95 @@ import { K_CACTUS_KEYCHAIN_MEMORY_TOTAL_KEY_COUNT } from "../../../main/typescri
+    ).toThrow();
+    expect(
       () =>
         new PluginKeychainMemory({
           instanceId: "",
           keychainId: "valid-value",
         }),
-    );
-    t.end();
+    ).toThrow();
   });
 
-  test("Validates constructor arg keychainId", (t: Test) => {
-    t.throws(
+  test("Validates constructor arg keychainId", () => {
+    expect(
       () =>
         new PluginKeychainMemory({
           instanceId: "valid-value",
           keychainId: null as any,
         }),
-    );
-    t.throws(
+    ).toThrow();
+    expect(
       () =>
         new PluginKeychainMemory({
           instanceId: "valid-value",
           keychainId: "",
         }),
-    );
-    t.end();
+    ).toThrow;
   });
 
-  test("get,set,has,delete alters state as expected", async (t: Test) => {
+  test("get,set,has,delete alters state as expected", async () => {
     const options: IPluginKeychainMemoryOptions = {
       instanceId: uuidv4(),
       keychainId: uuidv4(),
     };
     const plugin = new PluginKeychainMemory(options);
 
-    const expressApp = express();
-    expressApp.use(bodyParser.json({ limit: "250mb" }));
-    const server = http.createServer(expressApp);
-    const listenOptions: IListenOptions = {
-      hostname: "0.0.0.0",
-      port: 0,
-      server,
-    };
     const addressInfo = (await Servers.listen(listenOptions)) as AddressInfo;
-    test.onFinish(async () => await Servers.shutdown(server));
     const { address, port } = addressInfo;
     const apiHost = `http://${address}:${port}`;
-    t.comment(
-      `Metrics URL: ${apiHost}/api/v1/plugins/@hyperledger/cactus-plugin-keychain-memory/get-prometheus-exporter-metrics`,
-    );
-
     const config = new Configuration({ basePath: apiHost });
     const apiClient = new KeychainMemoryApi(config);
 
     await plugin.getOrCreateWebServices(expressApp);
 
-    t.equal(plugin.getKeychainId(), options.keychainId, "Keychain ID set OK");
-    t.equal(plugin.getInstanceId(), options.instanceId, "Instance ID set OK");
+    expect(plugin.getKeychainId()).toBe(options.keychainId);
+    expect(plugin.getInstanceId()).toBe(options.instanceId);
 
     const key1 = uuidv4();
     const value1 = uuidv4();
 
     const hasPrior = await plugin.has(key1);
-    t.false(hasPrior, "hasPrior === false OK");
+    expect(hasPrior).toBe(false);
 
     await plugin.set(key1, value1);
 
     const hasAfter1 = await plugin.has(key1);
-    t.true(hasAfter1, "hasAfter === true OK");
+    expect(hasAfter1).toBe(true);
 
     const valueAfter1 = await plugin.get(key1);
-    t.ok(valueAfter1, "valueAfter truthy OK");
-    t.equal(valueAfter1, value1, "valueAfter === value OK");
+    expect(valueAfter1).toBeTruthy();
+    expect(valueAfter1).toBe(value1);
 
     await plugin.delete(key1);
 
     const hasAfterDelete1 = await plugin.has(key1);
-    t.false(hasAfterDelete1, "hasAfterDelete === false OK");
+    expect(hasAfterDelete1).not.toBeTruthy();
+    expect(plugin.get(key1)).not.toResolve();
```

Comment:
@awadhana Need to `await` this one so that there is no race with the other assertions